tests/workflows/pw/test_base.py

```diff
@@ -137,3 +137,32 @@ def test_sanity_check_no_bands(generate_workchain_pw):
 
     calculation = process.ctx.children[-1]
     assert process.sanity_check_insufficient_bands(calculation) is None
+
+
+def test_set_max_seconds(generate_workchain_pw):
+    """Test that `max_seconds` get sets in the parameters based on `max_wallclock_seconds` unless already set."""
+    inputs = generate_workchain_pw(return_inputs=True)
```

Comment:
because the `generate_inputs_pw` does not return all the minimally required inputs needed for the workchain. If you look at `generate_workchain_pw` it needs to add the inputs to a namespace, and pop an input. I don't want to have to add these lines in every test.